src/expressions.rst

```diff
@@ -3100,6 +3100,20 @@ a :t:`tuple enum variant` or a :t:`type` that implements any of the
 The :t:`call operand` is subject to :t:`auto dereferencing` until a
 :t:`callee type` is found, otherwise this is a static error.
 
+:dp:`fls_c40C6rg6rGv6`
+An :t:`argument operand` of a :t:`call expression` matches a :t:`parameter` or
+:t:`field` of the :t:`callee type` when its position and the position of the
+:t:`parameter` or :t:`field` of the :t:`callee type` are the same. Such an
+:t:`argument operand` is a :dt:`matched argument operand`.
+
+:dp:`fls_Gr1ixJ9vFjUm`
+The :t:`type` of an :t:`argument operand` of a :t:`call expression` and the
```

Comment:
You probably meant "matched argument operand" here. Here is an alternative wording:

The type of a matched argument operand and the type of the corresponding function parameter or field shall be unifiable.